### ISSUE TITLE
Allow highlight modifiers on just the text portion of lines

### ISSUE THREAD

**Issue by mvllow** [jesseduffield/lazygit#1767]
**Is your feature request related to a problem? Please describe.**
Currently I rely on bold text for differentiating selected lines. Most of the time this is fine, but once you get lost it can take several seconds to realise what you are selecting.

**Describe the solution you'd like**

> This issue is focused on the use of `selectedLineBgColor` being set to `reverse`

Having a simpler version of `reverse` in the theme section would be ideal. The behaviour now is to invert _all_ colours on the selected line, including unicode characters such as the tree lines and folder arrows.

<img width="428" alt="Screen Shot 2022-02-09 at 10 45 46 AM" src="https://user-images.githubusercontent.com/1474821/153253299-e93ee8ff-4a44-4173-a8f7-9e7796841ad9.png">

There are potentially two issues in this screenshot. One being the highlighting around "Files" as that could be a different group. Ignoring that, though, the primary issue is the "M swatches.svelte" line. In my opinion, it would be lovely to only highlight "M swatches.svelte" and not the area before the text.

**Describe alternatives you've considered**
The reverse modifier does get the job done – contrast is much easier to see. Another solution is to use the underline option which also helps differentiate. The underline also underlines more than just the filename which can look messy as well, however.

**Additional context**
To summarise, being able to highlight (whether it be reverse or underline etc.) just the text portion of lines could clean up the UI quite a bit when trying to get a little more contrast.

An added bonus could be an additional character at the end of the line when reverse highlighting (to act as padding).

**Comment by jesseduffield** [comment]
@mvllow there's some discussion on this issue happening here: https://github.com/jesseduffield/lazygit/pull/1877
